pygbm/loss.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+
+# TODO: ABC for losses, regression losses and classification losses
```

Comment:
I am not sure we actually need an ABC in this case. We can just use duck typing. Let's only use a base class if there is some redundant code to factorize across loss functions.